stratum/hal/lib/tdi/dpdk/dpdk_hal.cc

```diff
@@ -207,14 +210,41 @@ ::util::Status DpdkHal::Run() {
   const std::vector<std::string> external_stratum_urls =
       absl::StrSplit(FLAGS_external_stratum_urls, ',');
   {
+    std::string log_output_str;
     ::grpc::ServerBuilder builder;
     SetGrpcServerKeepAliveArgs(&builder);
 
-    builder.AddListeningPort(FLAGS_local_stratum_url,
-                             ::grpc::InsecureServerCredentials());
-
-    for (const auto& url : external_stratum_urls) {
-      builder.AddListeningPort(url, ::grpc::InsecureServerCredentials());
+    if (FLAGS_grpc_open_insecure_ports) {
```

Comment:
Please add/update the README with necessary details on default mode and configuration needed for the same.

Reply:
Added to Networking-recipe/docs folder